### PR TITLE
Subject: [Pull Request] Completing Capstone Project: Islamic Seminar Website

### PR DESCRIPTION
**Pull request by Zafron047**
Create website using
Topic: Islamic Seminar

I have accomplished the following tasks:

- [ ] Added a home page to provide an overview of the seminar, including essential details and key highlights.

- [ ] Created an about page that offers comprehensive information about the seminar's purpose, objectives, and organizers.

- [ ] Implemented responsive design elements to ensure optimal user experience on both mobile and desktop viewports.

- [ ] Integrated a storage system to store and manage speaker information efficiently.

- [ ] Implemented dynamic functionality for pop-up windows, enabling attendees to view additional details about individual speakers and their sessions.

Kindly review my pull request at your earliest convenience. 
Thank you for your time and assistance. I look forward to your feedback and, if necessary, making any required adjustments to meet the project's expectations.

Best regards,
[Nohain Islam Zafron]

### CODE REVIEW

**Review by KanzaTahreem — COMMENTED** [review]
# INVALID ❌ 

Hi @Zafron047  👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌
- [ ] Unfortunately, you have submitted pull request with failing linters 😓Please make sure that all linters are passing and you can see a green tick mark✔✨

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Zafron047** [comment]
@KanzaTahreem Yes that was an honest mistake. I accidentally submitted before all the tests were finished.
I have fixed and submitted again.

Thanks for the attempt I appreciate your kind time. 😄

File: README.md [hunk 1]
```diff
@@ -1 +1,158 @@
...
+
+Your support and encouragement are highly appreciated and motivate me to continue working on projects like this.
+
+Thank you for your support!
+
+<br>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my sincere gratitude to Microverse for providing me with this opportunity to learn and grow as a developer. Their support and guidance have been invaluable throughout my journey.
```
Comment (DuaneDave) [inline README.md]: According to the project description, it is expected to give appropriate acknowledgements to the author of the design [Cindy Shin](https://www.behance.net/adagio07), in order not to come off as plagiarizing.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        <li class="header-items">
+          <i class="fa-brands fa-facebook"></i>
+        </li>
+        <li class="header-items">
+          <i class="fa-brands fa-twitter"></i>
+        </li>
+        <li class="header-items">English</li>
+        <li class="header-items">My Page</li>
+        <li class="header-items">Logout</li>
+      </ul>
```
Comment (DuaneDave) [inline index.html]: According to the design, the top bar doesn't have a bottom left and right rounded edge, kindly remove it and make it a regular square edge to make your design match the template. Also, consider respecting the correct background color which is #272a31

round

File: index.html [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+      <img src="./assets/images/seminar-logo.png" alt="" />
+
+      <div class="rights-msg">
+        <h4>
+          © 2023 Islamic Dawah Center - “The believer’s shade on the Day of
+          Resurrection will be his charity.” (Al-Tirmidhi).
+        </h4>
+      </div>
+    </footer>
+    <!-- Rights Ends -->
```
Comment (DuaneDave) [inline index.html]: To have a complete website as the template, consider making the footer visible on the mobile view.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Zafron047 👋, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Zafron047) [reply README.md]: Sincere apology for forgetting this part. Thanks a lot to Cindy Shin.
Acknowledgment added in readme file.

Complete.

File: index.html (hunk 2 above)
Comment (Zafron047) [reply index.html]: Border Radius remove.

Complete.

File: index.html (hunk 3 above)
Comment (Zafron047) [reply index.html]: As per your advice I made the footer also available in mobile version.

Complete

**Comment by Zafron047** [comment]
Dear reviewer,
I have made the necessary changes provided by the previous reviewer I hope the changes pass through. Thanks a lot for your kind effort to review my PR.

Best regards,
[Nohain Islam Zafron]

File: style.css [hunk 4]
```diff
@@ -0,0 +1,758 @@
...
+  /* Header end */
+
+  /* navbar starts */
+  .navigation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1% 1% 0;
+  }
+
```
Comment (levy002) [inline style.css]: - [OPTIONAL] Kindly make sure to add spacing around the navigation bar in order to match the provided Behance design.

  - ### YOUR DESIGN
  
    image

  - ### BEHANCE DESIGN
    
   image

File: style.css [hunk 5]
```diff
@@ -0,0 +1,758 @@
...
+    font-size: 0.6rem;
+    padding-top: 0.8rem;
+  }
+
+  /* Rights Starts */
+  .rigts {
+    display: flex;
+    justify-content: space-between;
+    border-top: 1px solid #3b3e4544;
+  }
```
Comment (levy002) [inline style.css]: - [OPTIONAL] Kindly make sure the `footer`  on the `index` mobile screen is not available in the Behance design.

   - ### YOUR DESIGN
   
     image

   - ### BEHANCE DESIGN
   
   image

**Review by levy002 — APPROVED** [review]
## Hello @Zafron047 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

  unnamed


## Highlights
- No linter error :heavy_check_mark: 
 
## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (Zafron047) [reply style.css]: Thanks for the kind effort reviewing my code. 
I have changed the `flex` property to `space-around` to make the change.

Change: complete; 😸

File: style.css (hunk 5 above)
Comment (Zafron047) [reply style.css]: After your feedback and As per the original design I have reversed back the footer section as `display: none;`.
Complete. 👍 

Thanks for the feedback have a nice day. 😄

**Pull request merged**